types/open-graph-scraper/index.d.ts

```diff
@@ -262,6 +268,8 @@ declare namespace run {
         url: string;
         /** Sets the options used by `validator.js` for testing the URL. */
         urlValidatorSettings?: URLValidatorSettings;
+        /** Add custom meta tags if there are no what you want in open graph properties type */
```

Comment:
Let's use the official description:


```suggestion
        /** Here you can define custom meta tags you want to scrape. */
```

Reply:
Thanks! :)